coalib/processes/Processing.py

```diff
@@ -341,7 +345,26 @@ def instantiate_processes(section,
         log_printer,
         ignored_file_paths=glob_list(section.get('ignore', "")),
         limit_file_paths=glob_list(section.get('limit_files', "")))
+
+    # This stores all matched files irrespective of whether coala is run
+    # only on changed files or not. Global bears require all the files
+    complete_filename_list = filename_list
+
+    # We need to compute the changed files so that the cache gets updated
+    # even if caching is disabled (in which case all files will be processed
+    # but the cache will be updated).
+    changed_files = get_changed_files(filename_list, cache)
+
+    # If caching is enabled then the local bears should process only the
+    # changed files.
+    if section.get('caching', False):
+        filename_list = changed_files
```

Comment:
can you log an `info` message here that the run is with caching enabled and log messages from previous runs may not appear?